core/server/worker/src/main/java/alluxio/worker/block/meta/DefaultStorageTier.java

```
@@ -98,7 +97,7 @@ private void initStorageTier(boolean isMultiTier)
         StorageDir dir = DefaultStorageDir.newStorageDir(this, i, capacity, reservedBytes,
             dirPaths.get(i), dirMedium.get(mediumTypeindex));
         mDirs.put(i, dir);
-      } catch (IOException e) {
+      } catch (AlluxioRuntimeException | IllegalStateException e) {
```

Feels like here you can probably just catch all exceptions.